src/main/java/uk/gov/pay/api/model/telephone/TelephonePaymentResponse.java

```diff
@@ -0,0 +1,154 @@
+package uk.gov.pay.api.model.telephone;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TelephonePaymentResponse {
```

Comment:
you could remove all the `JsonProperty` annotations and just annotate the class with `@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)`

Reply:
Done

Reply:
Not sure if you've pushed these changes?

Reply:
Apologies - I forgot to do this class it's done now